bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml

```diff
@@ -10,7 +10,88 @@ metadata:
           "metadata": {
             "name": "gatekeeper"
           },
-          "spec": null
+          "spec": {
+            "affinity": {
+              "podAffinity": {
+                "requiredDuringSchedulingIgnoredDuringExecution": [
+                  {
+                    "labelSelector": {
+                      "matchLabels": {
+                        "auditKey": "auditValue"
+                      }
+                    },
+                    "topologyKey": "topology.kubernetes.io/zone"
+                  }
+                ]
+              }
+            },
+            "audit": {
+              "auditChunkSize": 66,
+              "auditFromCache": "Enabled",
+              "auditInterval": "10s",
+              "constraintViolationLimit": 55,
+              "emitAuditEvents": "Enabled",
+              "logLevel": "DEBUG",
+              "replicas": 1,
+              "resources": {
+                "limits": {
+                  "cpu": "100m",
+                  "memory": "20Mi"
+                },
+                "requests": {
+                  "cpu": "100m",
+                  "memory": "20Mi"
+                }
+              }
+            },
+            "image": {
+              "image": "docker.io/openpolicyagent/gatekeeper:v3.5.2",
+              "imagePullPolicy": "Always"
+            },
```

Comment:
I'm unsure if we want this in the example, since it's a deprecated part of the CR spec. Do the scorecard tests require every possible field in the CR to be in the examples? If not, I think it would be more useful if the example was closer to what we are encouraging users to create.

Reply:
I'm happy to remove it from here as well as docs. Perhaps next release we remove it also from the console UI. The release after that we remove completely, including code?

Reply:
Where is it in the console UI?

Reply:
Separate PR for removing `image` https://github.com/gatekeeper/gatekeeper-operator/issues/180